Orange/widgets/unsupervised/owmds.py

```diff
@@ -861,9 +863,10 @@ def jitter(x, factor=1, rstate=None):
                 size_data = MinPointSize + size_data * point_size
             elif have_data and size_index > 0:
                 size_var = self.sizevar_model[size_index]
-                size_data = column(self.data, size_var)
+                size_data = column(self.data, size_var, dtype=float)
                 size_data = scale(size_data)
                 size_data = MinPointSize + size_data * point_size
+                size_data[numpy.isnan(size_data)] = 1  # maybe 0?
```

Comment:
OK, then the current value of 1 should be fine. It's already smaller than anything else.